rewrite design team principles (#300)
dxw/playbook: playbook.md

Designers at dxw follow these principles:

1. **Democratise the design process**

>    Take into account, and value highly, the input of others in the design process. While we are ultimately responsible for design in a delivery team, we don’t work in isolation.

Reviewer: Maybe we should follow GDS guidelines and not use negative contractions? For example this would become `we do not work in isolation`

1. **Value clarity in everything we do**

   Use plain language in interfaces, writing and conversation.

1. **Overlap with other disciplines**

   Work closely with people that have different skills to you, in the knowledge that different perspectives make our decisions stronger and elevate our work.

1. **Communicate constantly**

   Never go missing. Our teammates always understand what we’re doing and why, because we’re all aiming for the same outcome.

1. **Know we are not the user**

>    Don’t assume to know how users of our services think or act. We acknowledge that things such as genetics, upbringing, religious and geographical culture, and past experiences make us all different. We work closely with our user researchers to gain understanding and insight about the people that use the things we design. _“One accurate measurement is worth more than a thousand expert opinions.”_ ~ Grace Hopper

Reviewer: ```suggestion
   Do not assume to know how users of our services think or act. We acknowledge that things such as genetics, upbringing, religious and geographical culture, and past experiences make us all different. We work closely with our user researchers to gain understanding and insight about the people that use the things we design. _“One accurate measurement is worth more than a thousand expert opinions.”_ ~ Grace Hopper
```

1. **Be educators**

   Know that being open about our work, and helping others to understand our decisions and why we make them, benefits everybody.

1. **Be kind**

   Be tolerant, understanding, empathetic and compassionate. Look for opportunities to help and care for others. Putting people at the centre of everything will help you be a better designer.

1. **Embrace questions**

>    Understand that when people ask questions and give feedback it is to make something better. If we can’t articulate our decisions then we may not have made the best choice.

Reviewer: ```suggestion
   Understand that when people ask questions and give feedback it is to make something better. If we can not articulate our decisions then we may not have made the best choice.
```

1. **Ask questions**

>    Always ask questions and seek the truth;  when we understand the problem we can solve it. When things seem muddy take a step back and try and find the original intent or gaps of understanding. _“If I had an hour to solve a problem I’d spend 55 minutes thinking about the problem and five minutes thinking about solutions.”_ ~ Albert Einstein

Reviewer: ```suggestion
   Always ask questions and seek the truth;  when we understand the problem we can solve it. When things seem muddy take a step back and try and find the original intent or gaps in understanding. _“If I had an hour to solve a problem I’d spend 55 minutes thinking about the problem and five minutes thinking about solutions.”_ ~ Albert Einstein
```

1. **Always assume the best intent**

   We are passionate about what we do, but we are a team. When you have difficult conversations or feedback we should always assume that it is coming from a good place.

1. **Tell stories**

>    When we talk about our work and why we do what we do, don’t just state the facts. Inject some humanity into it; turn it into a story with a beginning, a middle,  and an end.

Reviewer: ```suggestion
   When we talk about our work and why we do what we do, do not just state the facts. Inject some humanity into it; turn it into a story with a beginning, a middle,  and an end.
```
